README.md

```diff
@@ -12,9 +12,35 @@ This deployment deploys an ephemeral Ceph container alongside Ember CSI and its
 ```
 $ git clone https://github.com/embercsi/ember-csi-kubevirt.git
 $ cd ember-csi-kubevirt/
-$ make cluster-up
-$ make deploy
-$ make demo
+$ make all
+```
+
+At this point we have  fully working cluster with KubeVirt and Ember-CSI deployed. We can now deploy VMs.
+
+```
+# Creating a virtual machine
+$ kubectl apply -f https://raw.githubusercontent.com/kubevirt/demo/master/manifests/vm.yaml
```

Comment:
This command only works for me if I replace kubectl with ./cluster/kubectl.sh  (same for all following commands).

Reply:
Since we are creating an alias, we can use that instead. The new commit uses this.